docs/index.md

```diff
@@ -1,9 +1,46 @@
 
 # Welcome to WheelNext
 
-## Introduction
-
-Content to be added here.
+## What is WheelNext and Why should you participate !?
+
+WheelNext is an open-source initiative (<https://github.com/wheelnext> & <https://wheelnext.dev/>) aiming to improve the
+user-experience in the Python packaging ecosystem, specifically around the scientific computing and machine/deep learning space.
+The current state of Python packaging was designed and implemented at a time before the increasingly complex needs these use-cases demand were prevalent.
+Through WheelNext, we aim to evolve Python packaging standards to better address the needs of this important segement, while retaining its effectiveness for the overall ecosystem.
+
+Many of the problems and gaps are summarized on the following website: <https://pypackaging-native.github.io/>
+Objective: To improve the state of packaging for Python native code, making it easier to build, distribute, and install
+Python packages with complex native dependencies.
+
+### Key Issues Identified:
+
+- **Complexity:** Due to the wide variety of microarchitectures, execution environments, and highly specialized dependencies, building and packaging native code in Python is complex and often inconsistent across different platforms.
+- **Tooling Fragmentation:** Multiple tools and standards exist, leading to confusion and inefficiency.
+- **Dependency Management:** Managing dependencies, especially for native code, is challenging.
+- **Installation Issues:** Users often face difficulties installing packages with native code due to environment
+specific issues.
+- **Lack of Standardization:** There is no single, standardized way to handle native packaging in Python.
+- **No Package Index Prioritization:** The main package installer `pip` is lacking ability to prioritize a given package
+over another.
+- **Lack of Symlink Support:** Very important to reduce package and better support native code. Yet it is lacking in the
+packaging standard.
+- **No ability to build a binary for a specific microarchitecture:** We are lacking the ability to describe with specificity the platform,
+microarchitecture, or other execution environment details is supported by a given binary (ARMv7, ARMv8, etc.)
+- **No clear path to support non-default package indexes:** Using private / corporate package indexes is common
+practice, yet the tooling ecosystem does not have a solid and reliable way to support these workflows.
+
+### Proposed Solutions:
```

Comment:
I suggest renaming to "Approach to Proposed Solutions", because they're more about the philosophy or abstract approach rather than concrete solutions.

Reply:
Agreed & Done